core/app/models/spree/user_last_url_storer.rb

```diff
@@ -26,6 +26,7 @@ def self.rules
     #  or its subclasses. The controller will be passed to each rule for matching.
     def initialize(controller)
       @controller = controller
+      Spree::Deprecation.warn("This class will be removed without replacement on the realease of Solidus 4.0")
```

Comment:
But are we still using this in core [here](https://github.com/solidusio/solidus/blob/a22723079f88387f53f72b503db875b9d97aa3f5/core/lib/spree/core/controller_helpers/auth.rb#L54)? If it's not called anymore, that method should be deprecated as well.

Reply:
Good point!
I also removed that method from solidus_starter_frontend and solidus_auth_devise!

Reply:
Thanks!